src/store/ChartState.js

```diff
@@ -411,6 +417,7 @@ class ChartState {
             this.stxx.scrollTo(this.stxx.chart, scrollToTarget, () => {
                 this.stxx.setMaxTicks(5);
                 this.stxx.micropixels = 0;
+                this.setIsChartScrollingToEpoch(false);
                 this.stxx.chart.isScrollLocationChanged = true; // set to true to draw markers
```

Comment:
@nazanin-binary it seems that we don't need it anymore since the markers are drawn after scrolling. Actually, after using `FastMarker` by Deriv-app this flag is not usable anymore.